apps/web/pages/api/auth/forgot-password.ts

```diff
@@ -15,13 +15,21 @@ const limiter = rateLimit({
 
 export default async function handler(req: NextApiRequest, res: NextApiResponse) {
   const t = await getTranslation(req.body.language ?? "en", "common");
+
   const email = z
     .string()
     .email()
     .transform((val) => val.toLowerCase())
     .parse(req.body?.email);
 
-  const { isRateLimited } = limiter.check(10, email); // 10 requests per minute
+  let ip = req.headers["x-real-ip"] as string;
+
+  const forwardedFor = req.headers["x-forwarded-for"] as string;
+  if (!ip && forwardedFor) {
+    ip = forwardedFor?.split(",").at(0) ?? email;
```

Comment:
FYI https://vercel.com/docs/concepts/edge-network/headers#x-forwarded-for there's warning that IP-address won't be forwarded if Vercel is behind a proxy.